wordpress-mobile/WordPress-iOS
# Updated Podfile to absorb changes to WordPressKit

Supports the resolution of [this issue](https://github.com/wordpress-mobile/WordPressKit-iOS/issues/86
https://github.com/wordpress-mobile/WordPressKit-iOS/issues/79) in `WordPressKit`.

I've designated this a Draft PR, unless or until the associated update to [`WordPressKit`](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/103) has been properly versioned.

To test:

- Branch should build.
- Tests should pass.
- "Smoke test" of network connectivity in the app should work.

Update release notes:
